ocrd_calamari/recognize.py

```diff
@@ -54,17 +62,30 @@ def process(self):
             pcgts = page_from_file(self.workspace.download_file(input_file))
 
             page = pcgts.get_Page()
-            page_image, page_xywh, page_image_info = self.workspace.image_from_page(page, page_id)
+            page_image, page_coords, page_image_info = self.workspace.image_from_page(
```

Comment:
Why change the variable names? I am even using the `xywh` convention from ocrd_tesserocr?

Reply:
Because xywh is not what this second member gives (anymore). It is still a dict with the necessary coordinates, but in a different form that cannot be used directly. Early adopters looked it up for `'x'` or `'w'` keys, but this fails since core 2.0 (for good reasons). Hence the rename (also, at least gradually, in ocrd_tesserocr).